jobs/config.json

```diff
@@ -11474,7 +11474,7 @@
       "--deployment=node",
       "--gcp-project=k8s-jkns-ci-node-e2e",
       "--gcp-zone=us-central1-f",
-      "--node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/image-config-beta.yaml",
```

Comment:
@krzyzacy, does this mean `k8sbeta`, which is currently 1.10?

Reply:
I don't know what `k8sbeta`means...even if it means 1.10, we'd need to rename this image config every time. Doesn't sound right to me.

Reply:
...is k8s switching to alpha/beta/stable channels? Regardless, the config is actually tied to 1.10 (not a channel), so making it explicit still seems better to me.

Reply:
a lot of test-infra jobs use these channels so that we don't need to rename anything, just update the file pointers to contain different releases and testgrid entries that show release #s. @krzyzacy did this, I'm not sure if it's documented and he's OOO today 😬

Reply:
But you'll need to rename this config once you move 1.10 to stable, no? I guess I still don't get how naming a config file beta would help  :-|

Reply:
@BenTheElder @krzyzacy 
I originally thought it might be testing some beta features. I see lots of 1.10 tests are using `beta`, which is confusing. I think we should use `k8sbeta` to make it clear.

Reply:
I agree it's not useful to naming the config this way.

If node e2e tests can be configured the same way as others, we can probably get rid of those image configs.